src/test/resources/com/google/jenkins/plugins/computeengine/gce-test-key.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "gce-jenkins",
+  "private_key_id": "07bb0766d086f4efe762a7873f73e4f04b6933fe",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDe1AagMeYXrT1v\nvgb6/nyCe+trZC6cwjQSQYuR3cY/yuGrrJcqJud8UOLVs3stxWjQDzhkV9A7DS+q\ndbDnt9uonBvs+IovI7q06tU13XTRoSUf8KVeXcf6g8fan8+5+bbtPSHQOgXfIikw\nXCJiwDpfSPVVnTpthDiV0XSJlbOf1dYZdQ3wJCpQorVTDgpFjqlOwRKBjJfqmg79\nZqgMB/1q6x2q7iMPF9miJZBUsMmTBGpvWBPt3NlvTiwaFW87MPcz8Fp1hlKpZTKo\ntnqqIAplhd+c0V25/NW76YUUiODGS45B7CBQx0hxoIIVNbFD3CsEsi3NHjQzIU2k\nWf58m7YFAgMBAAECggEANhvp/PoI++xhTBJquwXTodLogRcLe5zsDJfWhsYCF2kS\nej/nmECIu0cgAhE6ou96R3W+tZ0eT0yhK8U2/1nF8OuIhXeIBLDqLHcO7azYanQ1\nYedrnLnsrwBTplri1DAXk83huWDxnpC6zEe/QNgZhJxgmNjcgI4YOIc7g5Jy4yeT\nujY3C0OBRjuxOzRdv4wSiYCiN9cuWgyklIjAJvkiSdoIKVBiP0CbVhQNuNwpTtPA\nDpqXI8shAtVJtTJsvbWeqE+pUHl4luyutTEJEdCtMlcgeBRXHlibU1PagzdfTkps\noOuuXCD6huPE4HOtf3J35yTzdF4lEs+EDKe0XNWZGQKBgQDy12CMjPgqGz8diSyO\nhqwMZEIFZyuMhNJLRQF/XpKkfbwD+xTStq5bnL6MXUQtnky6ks9W3COgP8rPEevW\nk8t68Jl7+hCrQ2jJz4j0d1RNWXX5qSIL0XEEicfIY97PLhwVwhMWPBpObTEAwj8Z\nreXd+n9J7iJ4NU8035/ENlM4qQKBgQDq5wh2/O1YKwvPoCChbMrHpdd0FE/q63oZ\nJK2dwelTIrzlZKMXG8HhbEFhiIkr7X5vGMEoMn2nvyqPDmTNd395jbqaAEZa+n5Y\nBCLb3zlZh4z94X6RR/IifzBJHRI38w2AfyrOGzvBXinpdGGoo6HFMrCstt3jJ3JD\n8Oyb/2Zf/QKBgQDt0guV2gC3i9N1438lXbJkuFMH44/8n58rY7sht8a5HQHijCRK\nb+m1gn4CDNzWKC9QkLJJiHHDYHYH8K8ch8VA8xv9FFr6vpuC+KJReAOxdKlbBU54\ncHbUUPK8x0HPeIK8tEHpBpdksv7IlXMifPGfQNEj3/kV7G9TgqV+jQ1XcQKBgBMR\nfhZqobmuhN7t2LhB5JQgWkUKARLI2rh/ybgsrYwaHcthR/vwPOTXrNcGBlySgtjx\n4Xwp3Rlo40OJn5UuRiLP5n3p18w62Va7vzip/jXOUxuo2mVgub8jJABLY9c227m6\nmPbyLv+bbTuOPv85iUBAxSbAEXMHGLd7LznamazhAoGADkmFotJRYEJubHkL7IDl\nqXSRreakPZ1TqAFl3qLuXpHc9r74AqOxtcE3zMkI1CeTL97P10lKqE5GvrMLTnT3\n1tgOhqabHEC1xSYuNaWtvK3o4QcGvKB9p9cymjGk87QX7vFbwriSMZzHdZK/4CKU\nG5JEypiuYG/woE6R3IiiGG4=\n-----END PRIVATE KEY-----\n",
```

Comment:
Please remove this file. We don't want to be hosting creds in a public repo. The int tests use the GOOGLE_CREDENTIALS environment variable for specifying the test SA.

Also, I suggest rotating this key since it's been posted publicly.

Reply:
Ofc you know that this creds are fake :)
I would not put my credentials there..

And if I will use `GOOGLE_CREDENTIALS` variables I will have to move it to IT tests.. and I dont want to do that..
This is fake key for sake of creating (not connecting) client.

Reply:
Instead I suggest using a mock in the test. Even is this is an invalid key, it will get flagged by cred scanners which we'd prefer to avoid.

Reply:
Heck it.. so simple.. and I was fighting with it so long..
Exactly thats why peer reviews are needed.. :)
Thanks..

Reply:
Thank you as well, I was about to make the same mistake in the oauth plugin CasC tests